build.sbt

```diff
@@ -104,6 +104,7 @@ lazy val sapi             = Project("daffodil-sapi", file("daffodil-sapi")).conf
 lazy val tdmlLib             = Project("daffodil-tdml-lib", file("daffodil-tdml-lib")).configs(IntegrationTest)
                               .dependsOn(macroLib % "compile-internal", lib, io, io % "test->test")
                               .settings(commonSettings)
+                              .settings(libraryDependencies ++= Dependencies.core)
```

Comment:
Why is this change even needed?  You have `.settings(commonSettings)` on the previous line and that calls libraryDependencies ++= Dependencies.common, in short, core ++ infoset ++ test.

Reply:
I don't think this comment was addressed. I agree that this shouldn't be needed.